EDDiscovery/UserControls/Overlays/UserControlSurveyor.cs

```diff
@@ -409,9 +418,14 @@ private string InfoLine(ISystem sys, StarScan.ScanNode sn, JournalScan js)
             information.Append((js.Earthlike) ? @" is an earth like world.".T(EDTx.UserControlSurveyor_isanearthlikeworld) : null);
             information.Append((js.WaterWorld && !js.Terraformable) ? @" is a water world.".T(EDTx.UserControlSurveyor_isawaterworld) : null);
             information.Append((js.WaterWorld && js.Terraformable) ? @" is a terraformable water world.".T(EDTx.UserControlSurveyor_isaterraformablewaterworld) : null);
-            information.Append((js.Terraformable && !js.WaterWorld) ? @" is a terraformable planet.".T(EDTx.UserControlSurveyor_isaterraformableplanet) : null);
+            information.Append((js.PlanetTypeID == EDPlanet.High_metal_content_body && js.Terraformable) ? @" is a terraformable high metal content world." : null);
```

Comment:
these are missing those all very important .t(EDTX.. )

make up a new applicable ID in the .cs, and add a entry to the example-ex file (translation-example-uc.tlp) in the right section!

Reply:
Yes Sir!